modules/gapi/src/backends/fluid/gfluidimgproc.cpp

```diff
@@ -2088,32 +2188,44 @@ GAPI_FLUID_KERNEL(GFluidResize, cv::gapi::imgproc::GResize, true)
        }
        cv::Size outSize(outSz_w, outSz_h);
 
-       if (in.chan == 3)
+       if (in.depth == CV_8U && in.chan == 3)
        {
-           initScratchLinear<uchar, Mapper, 3>(in, outSize, scratch, LPI);
+           initScratchLinear<uchar, linear::Mapper, 3>(in, outSize, scratch, LPI);
        }
-       else if (in.chan == 4)
```

Comment:
but where is `chan == 4` at now?

Reply:
It is still not implemented.

Reply:
Then i'm confused a little bit: because in comparison with old/red code this case had been implemented but was broken in current PR and chan == 1 added instead

Reply:
Case `chan==4` wasn't implemented. For this case a stub was made. Stub was done because the plans were to implement this case immediately after adding case chan==3. But the plans were changed, but the stub remained. And now I have removed it. That's the whole story.

Reply:
ok, discussed offline